demos/editor/callbacks.py

```diff
@@ -28,7 +29,7 @@ def validate_positive(value):
     return min(0, value)
 
 
-def validate_color(color, default='#999999'):
+def validate_color(color: str, default='#999999'):
```

Comment:
this breaks compatibility with python 2, right?

Reply:
Yes.

Reply:
is it worth breaking python 2 compatibility just for this feature?

Reply:
No. I'll change it to `def validate_color(color='''', default='#999999'):` so that I do not get PEP8 errors because of erroneous type inference. Every other aspects of the app should be 2.7 compatible.

Reply:
Done.